sync.py

```diff
@@ -138,16 +140,17 @@ def yes_or_no(question):
 print('Syncing directories:\n{}\n{}'.format(src_root_path, src_containers_path))
 
 excludes = '--exclude=*.swp --exclude=irma3/'
-rsync_cmd = '/bin/rsync -avzP --relative --omit-dir-times {0} --log-file={1} {2} {3} {4}@{5}:{6}'.format(excludes,
+rsync_cmd = '/bin/rsync -avzP --relative --omit-dir-times {0} --log-file={1} {2} {3} {4} {5}@{6}:{7}'.format(excludes,
                                                                                                rsync_log_path,
                                                                                                src_root_path,
                                                                                                src_containers_path,
+																							   src_path_link,
```

Comment:
Looks like a couple of tabs too many here 😅

Reply:
Removed it, atom added extra tabs.